pkg/client/unversioned/request_test.go

```diff
@@ -729,6 +733,44 @@ func TestDoRequestNewWay(t *testing.T) {
 	fakeHandler.ValidateRequest(t, requestURL, "POST", &reqBody)
 }
 
+// This test assumes that the client implementation backs off exponentially, for an individual request.
+func TestBackoffLifecycle(t *testing.T) {
+	count := 0
+	os.Setenv(envBackoffBase, "1")
+	os.Setenv(envBackoffDuration, "120")
```

Comment:
I would prefer you create a request directly with the desired backoff instead of setting and unsetting env variables in tests.